browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2204,6 +2204,15 @@ Are you sure you want to do this?
     <message name="IDS_BRAVE_WALLET_RESET_SETTINGS_CONFIRMATION_PHRASE" desc="The phrase users should type to reset Brave Wallet">
       Yes
     </message>
+    <message name="IDS_BRAVE_WALLET_CLEAR_TX_AND_NONCE_SETTING_TITLE" desc="The title of a button that will reset transaction and nonce information. As in to erase the users transaction history and reset nonce value starting from 0x0">
+      Clear transaction &amp; nonce info
+    </message>
+    <message name="IDS_BRAVE_WALLET_CLEAR_TX_AND_NONCE_SETTING_DESC" desc="The footer message below the button to reset transaction and nonce">
+      Clearing transactions may be useful for developers or when clearing state on a local server
+    </message>
+    <message name="IDS_BRAVE_WALLET_CLEAR_TX_AND_NONCE_DIALOG_CONFIRMATION" desc="Confirmation dialog description for transaction and nonce reset.">
+      Are you sure you want to reset transaction and nonce information? This option is mostly used by developers running a local test server. Type "Yes" to confirm.
```

Comment:
`Yes` has to be a placeholder and replaced dynamically with `brave_wallet_reset_settings_confirmation_phrase`. Translators can translate `brave_wallet_reset_settings_confirmation_phrase` and the above `Yes` differently on some languages and there will be no way basically to know what should be the phrase for a user. Check how it's done for reset wallet
```
<message name="IDS_BRAVE_WALLET_RESET_SETTINGS_CONFIRMATION" desc="Confirmation for Brave Wallet reset.">
      Are you sure you want to reset your wallet? If your wallet is not backed up, resetting will cause you to lose all account data (including any associated funds). Type "<ph name="CONFIRMATION_PHRASE">%1$s<ex>Yes</ex></ph>" to confirm.
    </message>
```

Reply:
thanks, missed replacing that, will change.